.github/workflows/benchmarks.yml

```diff
@@ -1,3 +1,9 @@
+# Detray library, part of the ACTS project (R&D line)
+#
+# (c) 2024 CERN for the benefit of the ACTS project
+#
+# Mozilla Public License Version 2.0
+
```

Comment:
This might be a bit excessive, but ok :)

Reply:
Hey, I would put these into JSON files as well if I could... 😆